requirements_test.txt

```diff
@@ -3,7 +3,7 @@ docopt
 psutil
 requests
 ansible
-ansible-core~=2.16.0; python_version >= "3.11"
+ansible-core~=2.19.0b4; python_version >= "3.11"
```

Comment:
we can not merge this. If it is some thing temporal to later link to passed tests, ok. But even in that case could not be the best approach. Instead, we can update the CI pipeline and add a new item in the matrix, so we can install ansible 2.19 in a different run and be able to test both versions of ansible for this change and any other afterwards.